sessions/js-functions-1/vending-machine/js/index.js

```diff
@@ -0,0 +1,32 @@
+console.clear();
+
+/* --- || Vending Machine || --- */
+/* --- --- --- */
+/* --- --- --- */
+/* --- || Goal: Write a function that mimics the basic features of a vending machine. - */
+
+/* --- ! Read all of the tasks carefully before starting to code ! --- */
+
+/* 1. Write a function with the  name "vendingMachine". */
+
+/* 2. The function should include a "drink" parameter. */
+
+/* 3. In the function body, implement a console.log() that logs a string to the console 
+including the "drink" parameter, informing the user that they have received their drink successfully.
+The result should look something like this: "Here's your apple juice!" */
+
+/* 4. Call your function with a drink of your choice and check the console whether 
+your function words as intended. */
+
+/* Hint: You may need to think about how to combine your drink parameter and the rest of the string
+inside of your console.log() so that the result is a complete sentence. */
+
+/* Question: What happens if you call the vendingMachine function without an argument? */
+
+/* Bonus: 5. Find a way to assign a default value to a parameter in case the
+function is called without an argument. This needs to be implemented when you're declaring the function,
+not when you're calling it. */
+
```

Comment:
Maybe its better to move this into the readme, since the students are used to find the tasks in there at this point.